apps/spreedme.sh

```diff
@@ -29,6 +27,19 @@ root_check
 # Nextcloud 13 is required.
 lowest_compatible_nc 13
 
+# Check if spreedme is already installed
+print_text_in_color "$ICyan" "Checking if Spreed.me is already installed..."
+if [ -d "$SNAPDIR" ]
+then
+    msg_box "It seems like 'Spreed.me' is already installed.\nIf you continue, it will be reinstalled."
+    if [[ "no" == $(ask_yes_or_no "Do you really want to continue?") ]]
+    then
+        exit
+    fi
+fi
+
+print_text_in_color "$ICyan" "(Re-)Installing Spreed.ME..."
```

Comment:
Don't install spreed.ME, install Talk instead. Spreed.ME isn't even in the startup script anymore.

Reply:
So shall I remove all changed that I have made to spreed.me?

Reply:
all changes to spreed.me have been reverted.